README.md

```diff
@@ -80,12 +80,12 @@ Before you begin, please note that you will need to install or have available th
 
 -   [gcloud](https://cloud.google.com/sdk/docs/install)
 -   [gsutil](https://cloud.google.com/storage/docs/gsutil_install)
--   [Python 3.6+](https://www.python.org/downloads/) (Any Python version greater than or equal to 3.6)
+-   [Python 3.10+](https://www.python.org/downloads/) (Any Python version greater than or equal to 3.10)
```

Comment:
Wow. Is there a reason we need 3.10? Did I change a config somewhere?

Reply:
the backend setup requires 3.10. so it's not technically here for the cookiecutter but 3.10 is needed soon after 🤷‍♀️

Reply:
I can revert this change if you prefer

Reply:
Nope it makes sense for people to stay up-to-date!